lib/tasks/configure.rake

```diff
@@ -2,22 +2,35 @@ require "rake"
 
 # Rake tasks for making a vault cert
 namespace :configure do
+  desc "Make Vault, Astral, and OIDC provider certs"
+  task ssl: [ :vault_ssl, :astral_ssl, :oidc_provider_ssl ]
+
   desc "Make the server cert for vault"
-  task :ssl, [ :cert_name ] do |t, args|
-    cert_name = args[:cert_name]
-    cert_name = "vault" if cert_name.nil?
-    sanParam = "subjectAltName=DNS:#{cert_name}"
```

Comment:
The sanParam is actually needed for the oidc provider cert.  I mentioned it in my PR here: https://github.com/G-Research/astral/pull/49#issuecomment-2427657044

Reply:
will restore -- I assume can we use this in all cases?

Reply:
> I assume can we use this in all cases?

That is my understanding.  I did recreate the vault cert with it, and all the tests still pass with the address set like so: "VAULT_ADDR=https://vault:8443"